agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java

```diff
@@ -747,12 +748,12 @@ public void validate(ProcessorType processorType, IncludeExclude includeExclude)
     }
 
     private void validAttributeProcessorIncludeExclude(IncludeExclude includeExclude) {
-      if (attributes.isEmpty() && spanNames.isEmpty()) {
+      if (attributes.isEmpty() && spanNames.isEmpty() && logBodies.isEmpty()) {
```

Comment:
can we pass into the method whether it's a span or log processor, so we can validate and log more precisely?

Reply:
It's type Attribute which is applicable to both Log and Span. We don't know yet if it's log or span if you check the call stack.

Reply:
oh ya, got it 👍